nsxt/resource_nsxt_policy_gateway_redistribution_config.go

```diff
@@ -181,6 +181,12 @@ func resourceNsxtPolicyGatewayRedistributionConfigRead(d *schema.ResourceData, m
 	d.Set("bgp_enabled", config.BgpEnabled)
 	d.Set("ospf_enabled", config.OspfEnabled)
 	d.Set("rule", getLocaleServiceRedistributionRuleConfig(config))
+	if isPolicyGlobalManager(m) {
+		d.Set("site_path", getSitePathFromEdgePath(*obj.EdgeClusterPath))
```

Comment:
Are we sure EdgeClusterPath is set? Import can be requested for all sorts of weird objects.